src/sqs/service/SqsMessageService.ts

```diff
@@ -0,0 +1,123 @@
+import * as AWS from 'aws-sdk';
+
+import GetMessageOptions from './model/GetMessageOptions';
+import DecompressionService from '../../decompression/service/DecompressionService';
+
+const MAX_NUMBER_OF_MESSAGES = 10;
+
+export default class SqsMessageService {
```

Comment:
Is this class really needed by any consumer services? I thought all the inventory-consumer-services would only need SqsPollerService.

Please remove it if it is not needed.